functions/imagemagick/index.js

```diff
@@ -66,20 +66,20 @@ exports.blurOffensiveImages = (event) => {
 // [START functions_imagemagick_blur]
 // Blurs the given file using ImageMagick.
 function blurImage (file) {
-  const tempLocalFilename = `/tmp/${path.parse(file.name).base}`;
+  const tempLocalPath = `/tmp/${path.parse(file.name).base}`;
 
   // Download file from bucket.
-  return file.download({ destination: tempLocalFilename })
+  return file.download({ destination: tempLocalPath })
     .catch((err) => {
       console.error('Failed to download file.', err);
       return Promise.reject(err);
     })
     .then(() => {
-      console.log(`Image ${file.name} has been downloaded to ${tempLocalFilename}.`);
+      console.log(`Image ${file.name} has been downloaded to ${tempLocalPath}.`);
 
       // Blur the image using ImageMagick.
       return new Promise((resolve, reject) => {
-        exec(`convert ${tempLocalFilename} -channel RGBA -blur 0x24 ${tempLocalFilename}`, { stdio: 'ignore' }, (err, stdout) => {
+        gm(tempLocalPath).blur(16).write((tempLocalPath), (err, stdout) => {
```

Comment:
Why the parenthesis around `tempLocalPath`?

Reply:
[`gm` is a constructor](http://aheckmann.github.io/gm/docs.html#blur) - this specifies the target file path.